release/heat-map/heat-map.module.d.ts

```diff
@@ -1,2 +1,6 @@
+import { HeatMapCell } from "./heat-map-cell.component";
```

Comment:
Release files are still here. Please revert them to latest version in the master branch.